tests/simulation/test_backend.py

```diff
@@ -13,11 +13,11 @@
 )
 from tests.helpers.functions import (
     vector_almost_equal,
-    generate_random_matrix_products,
```

Comment:
@paolodr98 The additional comma serves the following purpose: When you add a new item to the list (a new import), then you wont have to add a comma on the second last line because there is already one. This means that you only have to change on line instead of two which is easier to understand.